src/microbe_stage/editor/CellEditorComponent.cs

```diff
@@ -1855,6 +1855,12 @@ private void OnOrganellesChanged()
 
         StartAutoEvoPrediction();
 
+        if (energyBalanceInfo != null)
+        {
+            TutorialState?.SendEvent(TutorialEventType.MicrobeEditorPlayerEnergyBalanceChanged,
```

Comment:
This triggering of this event should go in `UpdateEnergyBalance` as that is where the updated energy balance is handled, so that is where this logically belongs.